sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/json/JsonExpressionEvalUtils.scala

```diff
@@ -55,6 +55,10 @@ object JsonExpressionEvalUtils {
 
     UTF8String.fromString(dt.sql)
   }
+
+  def fromJson(jsonToStructsEvaluator: JsonToStructsEvaluator, json: UTF8String): Any = {
```

Comment:
why do we prefer `StaticInvoke` over `Invoke`? is there a feature gap?

Reply:
Just a personal preference, let me use `Invoke` to implement it.

Reply:
Updated.